src/loader.js

```diff
@@ -0,0 +1,68 @@
+/* eslint func-names: 0 */
+
+import resolveResources from './utils/resolveResources';
+import logger from './utils/logger';
+
+module.exports = function(source) {
+  if (this.cacheable) this.cacheable();
+
+  global.__DEBUG__ = (
+    process.env.DEBUG === 'sass-resources-loader' || process.env.DEBUG === '*'
+  );
+
+  logger.debug(`Hey, we're in DEBUG mode! Yabba dabba doo!`);
+
+  logger.debug('Context:', this.context);
+
+  const { sassResources } = this.options;
+
+  if (!sassResources) {
+    throw new Error(`
+      Could not find sassResources property.
+      Make sure it's defined in your webpack config.
+    `);
+  }
+
+  logger.debug('sassResources:', sassResources);
+
+  const sassResourcesIsString = typeof sassResources === 'string';
+  const sassResourcesIsArray = Array.isArray(sassResources);
+
+  logger.debug('sassResources is String:', sassResourcesIsString);
+  logger.debug('sassResources is Array:', sassResourcesIsArray);
+
+  if (!sassResourcesIsString && !sassResourcesIsArray) {
+    throw new Error(`
+      Looks like sassResources property has wrong type.
+      Make sure it's String or Array of Strings.
+    `);
+  }
+
+  const result = [];
+
+  if (sassResourcesIsString) {
+    const sassResourcesImport = resolveResources(sassResources, this);
+    logger.debug('Resources:', sassResourcesImport || '**not found**');
```

Comment:
I'm still not sure about this one, I think we should log it and throw (I'm not throwing here mainly b/c webpack will handle this throw in try/catch, and it's formatting is not cool for debug, but it noticeable).